projects/ecoindex_scraper/README.md

```diff
@@ -1,6 +1,6 @@
 # Ecoindex Scraper
 
-This module provides a simple interface to get the [Ecoindex](http://www.ecoindex.fr) of a given webpage using module [ecoindex-python](https://pypi.org/project/ecoindex/)
+This module provides a simple interface to get the [Ecoindex](http://www.ecoindex.fr) of a given webpage using module [ecoindex-compute](https://pypi.org/project/ecoindex-compute/)
```

Comment:
It would be beneficial to add a section on error handling. This can help users understand what to do when they encounter certain errors. [medium]